test/dkim/signed_mail_test.rb

```diff
@@ -20,7 +20,7 @@ def test_defaults
 
       # bh value from RFC 6376
       assert_equal '2jUSOH9NhtVGCQWNr9BrIAPreKQjO6Sn7XIkfJVOzv8=', dkim_header['bh']
-      assert_equal 'dQOeSpGJTfSbX4hPGGsy4ipcNAzC/33K7XaEXkjBneJJhv6MczHkJNsfmXeYESNIh5WVTuvE5IbnDPBVFrL+b3GKiLiyp/vlKO2NJViX4dLnKT/GdxjJh06ljZcYjUA+PorHvMwdu+cDsCffN8A7IhfVdsFruQr3vFPD0JyJ9XU=', dkim_header['b']
+      assert_equal 'QppIlbEcMAX4axIDBcTDYmr5UMS+qZygn6pcHzxw5glhBU0rDMy2bAPN1SqaQnx8pnpbaVtvS5YpkzYf5HOSARRZKerKat1XiN1MHrZzSL7gBUdDU++uGVcqq/CS8sEfUKBQtbAdWychFUx0EkPZrDJdYQZy/UEd+mx1UY4GNY4=', dkim_header['b']
```

Comment:
@oz I'm concerned that the wrapping is happening in the wrong place as from my understanding this should not actually impact the signature.

The [RFC](https://datatracker.ietf.org/doc/html/rfc6376#section-2.11) for DKIM-Quoted-Printable states that:
> Whitespace in the encoded text is ignored.  This is to allow
>          tags encoded using DKIM-Quoted-Printable to be wrapped as
>          needed.